### PR TITLE
added home page and kainos branding

### PR DESCRIPTION
**Pull request by KristinaGeddis**

### CODE REVIEW

**Review by AmamMcMam — DISMISSED** [review]

File: frontend/viewdir/template.njk [hunk 1]
```diff
@@ -4,6 +4,20 @@
...
+    {% endblock %}
+
+    <style>
+        .footer-style{
+            background-image: url('/images/footer.jpeg'); 
+            background-size: 100% 100%; 
+            width: 100%; 
+            height: 300px;
+        }
+    </style>
```
Comment (andrewwa-kainos) [inline frontend/viewdir/template.njk:20]: Let's put the styling in another css file under `src/public/something`

File: frontend/viewdir/template.njk [hunk 2]
```diff
@@ -16,8 +30,28 @@
         </ul>
         </div>
     </nav>
+
+    <body style="background-color: white">
```
Comment (andrewwa-kainos) [inline frontend/viewdir/template.njk:34]: Let's keep styling out of the html

File: frontend/package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "main": "routes.js",
   "scripts": {
-    "start": "nodemon src/routes.js",
+    "start": "nodemon src/routes.js -e njk",
```
Comment (andrewwa-kainos) [inline frontend/package.json:6]: Nice! I didn't know about this 👍

**Review by andrewwa-kainos — CHANGES_REQUESTED** [review]

File: frontend/viewdir/template.njk [hunk 4]
```diff
@@ -6,22 +6,14 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
```
Comment (andrewwa-kainos) [inline frontend/viewdir/template.njk:9]: We might want to anchor the url to the root with a preceding `/`. At the least it makes it consistent with the urls to the images

```suggestion
    <link rel="stylesheet" type="text/css" href="/css/style.css" />
```

**Review by andrewwa-kainos — APPROVED** [review]

**Review by AmamMcMam — APPROVED** [review]

**Pull request merged**